--- modules/org.eclipse.jdt.core/src/org/eclipse/core/internal/resources/Project.java.sav11	2007-11-06 14:22:38.000000000 +0100
+++ modules/org.eclipse.jdt.core/src/org/eclipse/core/internal/resources/Project.java	2007-11-06 14:23:05.000000000 +0100
@@ -12,7 +12,10 @@
 
 import java.util.*;
 import org.eclipse.core.internal.events.LifecycleEvent;
-import org.eclipse.core.internal.utils.*;
+//import org.eclipse.core.internal.utils.*;
+import org.eclipse.core.internal.utils.Assert;
+import org.eclipse.core.internal.utils.Messages;
+import org.eclipse.core.internal.utils.Policy;
 import org.eclipse.core.resources.*;
 import org.eclipse.core.resources.team.IMoveDeleteHook;
 import org.eclipse.core.runtime.*;
