--- modules/org.eclipse.jdt.core/src/org/eclipse/core/runtime/model/PluginPrerequisiteModel.java.sav5	2007-11-06 13:35:57.000000000 +0100
+++ modules/org.eclipse.jdt.core/src/org/eclipse/core/runtime/model/PluginPrerequisiteModel.java	2007-11-06 13:36:10.000000000 +0100
@@ -10,7 +10,7 @@
  *******************************************************************************/
 package org.eclipse.core.runtime.model;
 
-import org.eclipse.core.internal.runtime.Assert;
+import org.eclipse.core.internal.utils.Assert;
 
 /**
  * An object which represents the relationship between a plug-in and a
